.changeset/strange-parents-repair.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Do we need to add empty changeset for this PR?

Reply:
CI failes otherwise